ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,61 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace StrongPasswordCheck {
+   class Program {
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the password:");
+         string password = Console.ReadLine ();
+         if (isStrongPassword (password))
+            Console.WriteLine ("The password is strong.");
+         else {
+            if (!isStrongPassword (password)) {
+               Console.WriteLine ("The password is not strong.Please ensure the password has 6 characters");

```

Comment:
"The password is not strong" message is repeated. It can be printed once at the start of the else block